src/app/lib/config/services/hausa.ts

```diff
@@ -97,7 +97,7 @@ export const service: DefaultServiceConfig = {
         videoPlayer: 'Video player',
       },
       liveExperiencePage: {
-        liveLabel: 'Kai-tsaye',
+        liveLabel: 'Kai tsaye',
         liveCoverage: 'Rahoto kai-tsaye',
```

Comment:
we don't mind the hyphen in the 'live coverage' translation? Was it a grammar thing for that particular use of the words?